windows-observ-lib/targets.libsonnet

```
@@ -335,5 +334,84 @@ local lokiQuery = g.query.loki;
         'irate(windows_net_packets_sent_total{%(queriesSelector)s}[$__rate_interval])' % variables
       )
       + prometheusQuery.withLegendFormat('{{ nic }} transmitted'),
+
+    replicationPendingOperations:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'sum(windows_ad_replication_pending_operations{%(queriesSelector)s})' % variables
+      )
+      + prometheusQuery.withLegendFormat('Operations'),
+    directoryServiceThreads:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'sum(windows_ad_directory_service_threads{%(queriesSelector)s})' % variables
+      )
+      + prometheusQuery.withLegendFormat('Directory service threads'),
+    replicationPendingSynchronizations:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'sum(windows_ad_replication_pending_synchronizations{%(queriesSelector)s})' % variables
+      )
+      + prometheusQuery.withLegendFormat('Operations'),
+    ldapBindRequests:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'rate(windows_ad_binds_total{bind_method=~"ldap", %(queriesSelector)s}[$__rate_interval])' % variables
+      )
+      + prometheusQuery.withLegendFormat('{{ instance }}'),
+
+    ldapOperations:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'rate(windows_ad_directory_operations_total{origin=~"ldap", %(queriesSelector)s}[$__rate_interval])' % variables
+      )
+      + prometheusQuery.withLegendFormat('{{ instance }} - {{ operation }}'),
+
+    bindOperationsOverview:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'rate(windows_ad_binds_total{%(queriesSelector)s}[$__rate_interval])' % variables
+      )
+      + prometheusQuery.withLegendFormat('{{ instance }} - {{ operation }}'),
+
+    intrasiteReplicationTraffic:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'rate(windows_ad_replication_data_intrasite_bytes_total{%(queriesSelector)s}[$__rate_interval])' % variables
```

You can add * 8 here, or change units from bps to Bps on the panel level instead.

```suggestion
        'rate(windows_ad_replication_data_intrasite_bytes_total{%(queriesSelector)s}[$__rate_interval]) * 8' % variables
```